src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -2053,6 +2054,7 @@ shared_ptr<runtime::Executable>
         case OP_TYPEID::Gather:
         case OP_TYPEID::GatherND:
         case OP_TYPEID::GenerateMask:
+        case OP_TYPEID::GRN:
```

Comment:
Please add `case OP_TYPEID::GRN:` line into is_supported_impl function switch (lines 2163 - 2172 of this file)